src/test/java/uk/gov/digital/ho/hocs/queue/common/integration/MigrationQueueListenerIntegrationTest.java

```diff
@@ -11,9 +11,9 @@
 
 import static org.springframework.boot.test.context.SpringBootTest.WebEnvironment.RANDOM_PORT;
 
-@SpringBootTest(webEnvironment = RANDOM_PORT, properties="case-creator.mode=migration")
+@SpringBootTest(webEnvironment = RANDOM_PORT)
 @RunWith(SpringRunner.class)
-@ActiveProfiles("local")
+@ActiveProfiles({"local", "migration"})
 public class MigrationQueueListenerIntegrationTest extends AwsSqsIntegrationTestBase {
```

Comment:
Might be good to look into testcontainers now for these kind of tests since we are on GitHub Actions

Reply:
OOS for now